solutions/instances/outputs.tf

```diff
@@ -64,6 +64,21 @@ output "scc_workload_protection_access_key" {
   sensitive   = true
 }
 
+output "scc_profile_attachment_id" {
```

Comment:
I think it is better to have one output with all info, since it is a part of the same object, like you suggested
```
scc_profile_info = [
  {
    "attachment_id" = "6c46eede-7f25-4514-8172-a722f6ae7e72/fe96bd4d-9b37-40f2-b39f-a62760e326a3/386f7fe8-6a64-4a84-97b2-ed8c7e5b07c6"
    "name" = "IBM Cloud Framework for Financial Services"
    "version" = "1.7.0"
  },
]
```

Reply:
Thank you @akocbek - I have made change to have one output block.